src/abstracts/SablierV2Lockup.sol

```diff
@@ -421,9 +421,11 @@ abstract contract SablierV2Lockup is
         notNull(streamId)
         returns (uint128 withdrawnAmount)
     {
-        // Check: the caller is the current recipient. This also checks that the NFT was not burned.
+        // Check: NFT exists and retrieve the current owner.
```

Comment:
Since we are not "checking" anything on the next line.

```suggestion
        // Retrieves the current owner. This also checks that the NFT was not burned.
```